defaults/main.yml

```diff
@@ -1,6 +1,14 @@
 ---
 # defaults file for OULibraries.dspace6
 
+# Global Tomcat vars
+dspace_tomcats_base: /var/lib/tomcats/
```

Comment:
`/var/lib/tomcats` seems like a typo? Also, I think some of these paths are baked in to the RPMs and we probably don't need variables for them until we start doing our own tomcat install.

Reply:
This is an unfortunate config on their end and what it is listed as in production

Reply:
Fixed the typo